src/ray/core_worker/test/mock_worker.cc

```diff
@@ -36,6 +39,27 @@ class MockWorker {
     // Note that this doesn't include dummy object id.
     RAY_CHECK(num_returns >= 0);
 
+    if (ray_function.function_descriptor.size() == 1 &&
+        ray_function.function_descriptor[0] == c_actor_creation_function_str) {
```

Comment:
it's better to tell actor creation tasks by `TaskInfo.task_type`

Reply:
makes sense

Reply:
this is not only for actor creation task, but has a special meaning that the caller would pass in a list of object IDs via `args`, is used to test actor FO. This is not the same as `TaskInfo.task_type`, the original naming and comment here are misleading, I've changed `c_actor_creation_function_str` to `c_test_actor_failover_str`